app/src/main/java/com/example/myapplication/SecondFragment2.java

```diff
@@ -407,10 +333,32 @@ public void onDestroyView() {
         binding = null;
     }
 
-    /**
-     *
-     * @param v current ViewGroup obtained from onCreateView
-     * @param mode false for light mode, true for dark mode
-     */
+    public void humanOperation(boolean checked, float width, float height){
+        int layout = checked ? 500:-38;
+        int view = checked? 500: 0;
+        int vis = checked? VISIBLE:GONE;
+        ColorStateList col = checked? ColorStateList.valueOf(UIHelpers.purple): ColorStateList.valueOf(UIHelpers.teamColor);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            binding.human.setThumbTintList(col);
+            binding.human.setTrackTintList(col);
+        }
+        ViewGroup.LayoutParams layoutParam = binding.relativeLayoutFirst.getLayoutParams();
+        layoutParam.width = (int) width;
```

Comment:
Do you need  this operation right here? It doesn't look like the value of width is ever changed